core/src/apps/bitcoin/sign_tx/matchcheck.py

```diff
@@ -107,6 +107,18 @@ def attribute_from_tx(self, txio: TxInput | TxOutput) -> Any:
         return multisig.multisig_fingerprint(txio.multisig)
```

Comment:
Since we removed the logic which accepts singlesig change outputs even when multisig inputs are present, we can simplify the code a little bit by doing the following here:
```py
if not txio.multisig:
    return bytes()
```
What this would do is allow us to:
1. get rid of the special multisig handling in `ChangeDetector.def output_is_change()`, where we only check the fingerprint for multisig outputs, and
2. get rid of the `MultisigChecker` altogether.

Reply:
I'm generally fan of explicit code but this seems fine, especially if you add a comment.

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/4351/commits/b7d26c794b8839fb9a35208d3b7a4ae336424480.